### ISSUE TITLE
Restore pretrained models error

### ISSUE THREAD

**Issue by benlansdell** [AdalbertoCq/Pathology-GAN#12]
I see this has been brought up in previous issues, but it is still an issue for me. I am unable to restore the pre-trained breast cancer model. 

(I can, however, restore the CRC model.)

I get the following error when I run this command: 
`
python generate_fake_samples.py --checkpoint ./pretrained_models/breast_cancer/PathologyGAN.ckt --z_dim 200 --batch_size 64
`

```
NotFoundError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key generator/attention_block_2/conv_layer_1/bias not found in checkpoint
         [[node save/RestoreV2 (defined at /home/blansdell/projects/comet/Pathology-GAN/models/evaluation/features.py:196)  = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
```

I'm using the code in the most recent commit (00b0736), which I see includes the fix suggested by issue (#10). Tensorflow version 1.12.0.

**Comment by WolfgangGross** [comment]
Maybe try --z_dim  128
From the code (https://github.com/AdalbertoCq/Pathology-GAN/blob/master/run_pathgan.py#L40) I'm getting this setting was used.

**Comment by benlansdell** [comment]
Thanks for the suggestion. I get the same error, however.

**Comment by micdic** [comment]
HI there, where did you find the pretrained model in that format?  (*.ckt). In the link provided i've found only the old TF checkpoint version  (PathologyGAN.ckt.data-00000-of-00001).
Thanks in advance.

**Comment by micdic** [comment]
The checkpoints downloaded cause this error, when restored:
Data loss: Unable to open table file checkpoints/PathologyGAN.ckt.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

**Comment by AdalbertoCq** [comment]
> I see this has been brought up in previous issues, but it is still an issue for me. I am unable to restore the pre-trained breast cancer model.
> 
> (I can, however, restore the CRC model.)
> 
> I get the following error when I run this command:
> `python generate_fake_samples.py --checkpoint ./pretrained_models/breast_cancer/PathologyGAN.ckt --z_dim 200 --batch_size 64`
> 
> ```
> NotFoundError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:
> 
> Key generator/attention_block_2/conv_layer_1/bias not found in checkpoint
>          [[node save/RestoreV2 (defined at /home/blansdell/projects/comet/Pathology-GAN/models/evaluation/features.py:196)  = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
> ```
> 
> I'm using the code in the most recent commit ([00b0736](https://github.com/AdalbertoCq/Pathology-GAN/commit/00b073666e72c863ae3dec0761d78228696d8fd3)), which I see includes the fix suggested by issue (#10). Tensorflow version 1.12.0.

Hi Ben, 

I think this is due to a difference between the scope naming for the generator/attention layer and the version for the model weights (Need to update this). Could you try replacing `scope=layer` by  `scope=layers` in here? https://github.com/AdalbertoCq/Pathology-GAN/blob/00b073666e72c863ae3dec0761d78228696d8fd3/models/generative/generator.py#L164

Thanks,
Adal

**Comment by benlansdell** [comment]
Hi Adal, 

Thanks for responding. Yes, that seems to fix the problem!

Cheers,
Ben

**Comment by AdalbertoCq** [comment]
Awesome, let me know if you run into any other issues.

Thanks,
Adal
